src/components/ui/Button/Button.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import PropTypes from "prop-types";
+import './Button.css'
+
+const  TFButton = (props) => {
+    const {icon, label, disabled, variant, handleClick, size, customStyles} = props;
+    const mainStyle={};
+    const textStyle = {};
+    const buttonClassName = variant==='secondary' ? disabled? 'secondary-btn-disabled' : 'secondary-btn': disabled ? 'primary-btn-disabled': 'primary-btn';
```

Comment:
THis needs to be a useState variable if we want to update it in order to avoid re rendering look at the chip component how dynamic css variables are created